Login UI

sanchit-sinha opened the pull request:
![invalid_email](https://user-images.githubusercontent.com/68417985/121063392-dd587e80-c7e3-11eb-9ff1-a6f02c390609.png)
![valid_email](https://user-images.githubusercontent.com/68417985/121063396-de89ab80-c7e3-11eb-94cd-52a0214df704.png)

mridul911 commented on src/views/examples/Login.js:
```diff
@@ -31,60 +31,55 @@ import {
+    var class_valid = "is-valid form-control";
+    var class_invalid = "is-invalid form-control";
+
+    target_email.className = class_invalid;
+    if (
+      /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@iitk.ac.in/.test(target_email.value)
+    ) {
```
Great!

mridul911 reviewed (approved):
Great work! Can you just remove some unnecessary things from login page and squash all commits into one.

sanchit-sinha commented:
Updated Login UI as per https://github.com/SPO-Career-Portal/SCP-Frontend/pull/9#pullrequestreview-677819144
![updatedui](https://user-images.githubusercontent.com/68417985/121151725-3e269c00-c862-11eb-8745-85c67eb62074.png)

kartikcode commented:
Remove the option of Create New Account. Also, maybe we can have SPO logo and title instead of the Argon one.

sanchit-sinha commented:
Hey, we are using the `username` and the `password` of the user as the POST request's credential(in the backend). So, shall I update the Login UI with the following changes:

- Using the `IITK Username` field instead of the `IITK Email ID`, for logging in  
- Removing the  Email validator
- Updating the dummy API

Suggest some other changes if required!

sanchit-sinha commented:
Made required changes as per https://github.com/SPO-Career-Portal/SCP-Frontend/pull/9#issuecomment-857786291 and https://github.com/SPO-Career-Portal/SCP-Frontend/pull/9#issuecomment-859839392
![Screenshot from 2021-06-13 11-24-54](https://user-images.githubusercontent.com/68417985/121797925-e3fe5000-cc40-11eb-82f9-52f8b0ea7e67.png)
closes #9

sanchit-sinha commented:
I have removed unnecessary information from the footbar, navbar and added a temporary logo. Do I need to revert these changes as per #10 or is it fine ??
